doc/getting-started.adoc

```diff
@@ -83,14 +83,34 @@ $ git branch -vv
 
 === Configure Jenkins
 
-You should have a Jenkins agent already created.
+Jenkins after the first startup of Jenkins container you will have to finish the install in your browser.
```

Comment:
```suggestion
Jenkins after the first startup of Jenkins container you will have to finish the install in your browser.
```
After the first startup of Jenkins container you will have to finish the install in your browser.

Reply:
you should have putted your suggestion at the right place to let me accept it ;-)
```suggestion
After the first startup of Jenkins container you will have to finish the install in your browser.
```